<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QWidget" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_player2_avatar">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_player2_name">
         <property name="text">
          <string>Nickname</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_player2_points">
         <property name="text">
          <string>Punts</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cerrarPushButton2">
         <property name="text">
          <string>Cerrar Sesión</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyProfileButton2">
         <property name="text">
          <string>Modificar Perfil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="modifyProfileButton1">
         <property name="text">
          <string>Modificar Perfil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cerrarPushButton">
         <property name="text">
          <string>Cerrar Sesión</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_player1_points">
         <property name="text">
          <string>Punts</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_player1_name">
         <property name="text">
          <string>Nickname</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_player1_avatar">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <layout class="QHBoxLayout" name="caca">
       <property name="topMargin">
        <number>20</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
